migrations/V186__Add_MTO_Solution.sql

```diff
@@ -18,12 +18,14 @@ CREATE TABLE mto_solution(
     name ZERO_STRING,
     -- we allow null because this is will be from the commonSolution table if it exists
     type MTO_SOLUTION_TYPE,
-    facilitated_by MTO_FACILITATOR,
+    facilitated_by MTO_FACILITATOR[],
     needed_by TIMESTAMP WITH TIME ZONE,
     status MTO_SOLUTION_STATUS NOT NULL,
     risk_indicator MTO_RISK_INDICATOR NOT NULL DEFAULT 'ON_TRACK',
-    poc_name ZERO_STRING NOT NULL,
-    poc_email EMAIL NOT NULL,
+    -- we allow null because this is is only required for custom solutions
+    poc_name ZERO_STRING,
+    -- we allow null because this is is only required for custom solutions
+    poc_email EMAIL,

```

Comment:
You know, I'm not sure. I considered it, but also thought we might want to verify when Zoë returns if this is a field that users will be able to set or not. I left it for now since it isn't clear.

Reply:
My guess is that anything that comes from the Common Milestone / Solution _probably_ isn't going to be editable, with the exception of Category?

I'm fine to leave this for now, though! Could you maybe flag a `TODO` to come back to this and add the constraint if we determine that to be the case?